scheduler/src/cook/mesos/api.clj

```diff
@@ -1116,22 +1119,31 @@
   (mapv (partial fetch-job-map (db conn) framework-id) (::jobs ctx)))
 
 (defn render-jobs-for-response
+  "This rendes for response. Fills in in :group UUID's and names as well as map-ifies jobs
+  It will examine the ctx for ::jobs (containing UUID's) or ::job-entities (containing
+  datomic job entities) and return the merged set."
```

Comment:
Why do we need both `::jobs` and `::job-entities`?

Reply:
::jobs has UUID's (coming from the /job/<UUID> and from /job&uuid=XXXX&uuid=XXXX) and ::job-entities has entities. Both have different logic as they filter through read-jobs-handler, so are separated until the very end.

Reply:
Is it possible to have them work the same way so that we don't need both entries in the context map?

Reply:
Yes, if we refactor the /jobs API endpoint implementation. It'll take time for me to get up to speed on liberator's semantics, because I'm going to have to face the ::params ::query-params, etc. All of the filtering and other functions in the &uuid=XXXX are all wired to UUID's. And if we're going to do that, we could greatly simplify it the implementation & semantics with a small API change.

Do we want really want to rescope this PR that much?

Reply:
> Thus the proposal for a change.

It's not clear to me what the change is that you're proposing.